homework17/js/main.js

```diff
@@ -0,0 +1,212 @@
+            || weather.Day.IconPhrase === 'Mostly Cloudy w/ Showers' || weather.Day.IconPhrase === 'Partly Sunny w/ Showers') {
+            img.src = '../assets/img/shower.png';
+        } else if(weather.Day.IconPhrase === 'Intermittent clouds' || weather.Night.IconPhrase === 'Intermittent clouds') {
+            img.src = '../assets/img/part-cloudy.png';
+        } else if(weather.Day.IconPhrase === 'Sunny' || weather.Day.IconPhrase === 'Mostly sunny'
+            || weather.Day.IconPhrase === 'Clear') {
+            img.src = '../assets/img/sun.png';
+        } else if(weather.Day.IconPhrase === 'Snow' || weather.Day.IconPhrase === 'Flurries' || weather.Day.IconPhrase === 'Freezing rain'
+            || weather.Day.IconPhrase === 'Rain and snow') {
+            img.src = '../assets/img/snowing.png';
+        } else if(weather.Day.IconPhrase === 'T-Storms') {
+            img.src =  '../assets/img/storm.png';
+        } else if(weather.Day.IconPhrase === 'Ice' || weather.Day.IconPhrase === 'Freezing Rain'
+            || weather.Day.IconPhrase === 'Cold'){
+            img.src = '../assets/img/freezy.png';
+        } else if(weather.Day.IconPhrase === 'Hot' || weather.Night.IconPhrase === 'Hot') {
+            img.src = '../assets/img/hot.png';
+        } else if(weather.Day.IconPhrase === 'Windy' || weather.Night.IconPhrase === 'Windy') {
+            img.src = '../assets/img/wind.png';
+        }
+
+        //Додаємо температуру
+        const temperature = document.createElement('p');
+        temperature.classList = 'weather__temperature';
+        temperature.innerHTML = `<span class="weather__title">Minimum:</span> ${temperatureConversion(weather.Temperature.Minimum.Value)} °C <br> 
+                                         <span class="weather__title">Maximum:</span> ${temperatureConversion(weather.Temperature.Maximum.Value)} °C`;
+
+        //Додаємо тип погоди
+        const phase = document.createElement('p');
+        phase.classList = 'weather__phase';
+        phase.innerHTML = `<span class="weather__title">Phase in night:</span> ${weather.Day.IconPhrase}<br> 
+                                    <span class="weather__title">Phase in day:</span> ${weather.Night.IconPhrase}`;
+
+        //Обираємо потрібний нам блок
+        const divInformation = document.querySelector('.weather__information');
+        //Додаємо всі елементи в блок, а потім до одного спільного блоку
+        const divBlockOneDay = document.createElement('div');
+        divBlockOneDay.classList = 'weather__information-block';
+        divBlockOneDay.append(dateElement, img, temperature, phase);
+        divInformation.append(divBlockOneDay);
+    }
+}
+
+//Створення сусідів
+const buttonsNeighbors = document.querySelector(".buttons");
+
+//Додаванняя інформації до кнопок
+const appendButtonNeighbors = (name, key) => {
+    const btnNeighbor = document.createElement("button");
+    btnNeighbor.classList = "button-neighbors";
```

Comment:
Try to use the same quotes
```suggestion
    btnNeighbor.classList = 'button-neighbors';
```